legacy/src/main/java/org/opensearch/sql/legacy/utils/Util.java

```diff
@@ -41,6 +43,19 @@ public class Util {
 
   public static final String NESTED_JOIN_TYPE = "NestedJoinType";
 
+  public static final Set<String> META_FIELDS_BEFORE_7DOT8 =
```

Comment:
1. Can we wrap this with a static method so we can change implementation easily?
2. If we go ahead with this approach, please add comment regarding this is not ideal solution.
3. What does the suffix `7D0T8` mean?

Reply:
7DOT8 is referring to ES 7.8.

"_ttl" and "_timestamp" were removed in ES6

"_type" was renamed to "_nested_path" in OS 2: https://github.com/opensearch-project/OpenSearch/pull/3196

"_size" is not a built-in metadata mapper, it comes from the mapper-size plugin which is not installed by default. Using mapperService.isMetadataField("_size") should return true if the mapper-size plugin is installed in the cluster.

The other values in this list still exist and are built-in metadata fields so mapperService.isMetadataField(<field>) should return true.